.env.prod

```diff
@@ -0,0 +1,8 @@
+EOS_SCATTER_PROTOCOL='http'
```

Comment:
I do not see IPFS url in .env file. Please add.
Do we have any other values that need to be moved to .env? Please check